nextpnr.dockerfile

```diff
@@ -12,10 +12,9 @@ RUN apt-get update -qq \
 FROM build AS build-ice40
 COPY --from=hdlc/pkg:icestorm /icestorm/usr/local/share/icebox /usr/local/share/icebox
 
-RUN mkdir -p /tmp/nextpnr/build \
- && cd /tmp/nextpnr \
- && curl -fsSL https://codeload.github.com/YosysHQ/nextpnr/tar.gz/master | tar xzf - --strip-components=1 \
- && cd build \
+RUN git clone --depth 1 https://github.com/YosysHQ/nextpnr.git /tmp/nextpnr \
```

Comment:
With `--depth 1` tags are not retrieved, are they? Some projects show the version as a difference from the last tag, plug the SHA of the current commit. See, for example `GHDL 1.0-dev (v0.37.0-1123-g8ed35277) [Dunoon edition]`. "Shallow clones" don't work in those cases.